### ISSUE TITLE
Not able to pass custom object instead of title only

### ISSUE THREAD

**Issue by iOSUser110** [ElaWorkshop/TagListView#132]
If am going to delete a tag , how can i maintain the id related to that tagView (title is only to display name ,but int is uniquely depend on its id)
Need solution for this very urgent..

**Comment by rabia-onschedule** [comment]
yes - at least we should be able to save id with title for tags.
